### PR TITLE
CSS Library - Utilities - Swap background-color and color imports 

### PR DESCRIPTION
**Pull request by rmessina1010**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Are you removing, renaming or moving a folder in this PR?
- [ ] No, I'm not changing any folders (skip to Summary and delete the rest of this section)
- [ ] Yes, I'm removing, renaming or moving a folder

If the folder you changed contains a `manifest.json`, search for its `entryName` in the content-build [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) (the `entryName` there will match).

If an entry for this folder exists in content-build and you are:
1. **Deleting a folder**: Delete the application entry in [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json) and merge that PR **before** this one
- _Add the link to your merged content-build PR here_

2. **Renaming or moving a folder**: Update the entry in the [registry.json](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/applications/registry.json), but do not merge it until your vets-website changes here are merged. The content-build PR must be merged immediately after your vets-website change is merged in to avoid CI errors with content-build (and Tugboat).

### :warning: TeamSites :warning:
Examples of a TeamSite: https://va.gov/health and https://benefits.va.gov/benefits/. This scenario is also referred to as the "injected" header and footer. You can reach out in the `#sitewide-public-websites` Slack channel for questions.

Did you change site-wide styles, platform utilities or other infrastructure?
- [ ] No
- [ ] Yes, and I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#that-sounds-normal-so-whats-the-proxy-all-about) to test the injected header scenario

## Summary

- _(Summarize the changes that have been made to the platform)_
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)
[https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3060](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/3060
)
## Testing done

- _Describe what the old behavior was prior to the change_
- _Describe the steps required to verify your changes are working as expected_
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |       |
| Desktop |        |       |

## What areas of the site does it impact?

*(Describe what parts of the site are impacted **if** code touched other areas)*

### Quality Assurance & Testing

- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [ ] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by longmd — DISMISSED** [review]

**Review by randimays — DISMISSED** [review]
I reviewed `resources-and-support/components/SearchBar` and found a few issues:

## Desktop Header
Contact us link is purple? I thought maybe the link was visited on my end but it doesn't appear to be
Screenshot 2024-08-14 at 8 53 27 AM

## Mobile Header
VCL text is blue
Screenshot 2024-08-14 at 8 53 37 AM

## Mobile Footer
VCL text is blue
Screenshot 2024-08-14 at 8 53 54 AM

## Mobile Resources & Support expandable search bar (`/resources`)
Button should be gray with blue text and icon. Here's the review instance:
Screenshot 2024-08-14 at 8 53 42 AM

This is what's in prod:
Screenshot 2024-08-14 at 8 53 47 AM

## Injected Header/Footer
Make sure to verify the injected header/footer on desktop and mobile. My team are handing this off to DST; we will not own it anymore. Check [this file](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/proxy-rewrite/proxy-rewrite-whitelist.json) for other domains (look for the `cookieOnly: false` value) that use the injected header.

Injected header on mobile on the review instance: Note that background behind the logo is a lighter blue than in prod (https://www.va.gov/health) and the dropdowns are different colors
Screenshot 2024-08-14 at 9 00 35 AM

Here's what prod looks like.
Screenshot 2024-08-14 at 9 00 45 AM

**Comment by rmessina1010** [comment]
As the scope of this ticket is to swap the name space of 2 colors.  I am running a couple of quick experiments. 
1) I will forgo the Formation override via css library's utilities.css
(If that works then we can conclude said file is not ready for prod)
2) Revert the formation bump to 11.0.13
(if that works , then  the error may be in the changes made to formation itself)

**Comment by randimays** [comment]
@rmessina1010 Just a heads up that I'll be on PTO for 2 weeks starting 8/20. If you need me to re-review, I'm available through Monday, 8/19.

**Review by alex-morgun-1 — DISMISSED** [review]

**Review by gabezurita — DISMISSED** [review]
The ARF owned `src/applications/accredited-representative-portal/components/common/Header/MobileHeader/MobileLogoRow.jsx` change LGTM!

**Review by chriskim2311 — DISMISSED** [review]
LGTM, tested changes in RI and injected header locally.

**Review by gabezurita — DISMISSED** [review]
The `accredited-representative-portal` LGTM!

File: src/applications/mhv-secure-messaging/sass/secure-messaging.scss [hunk 1]
```diff
@@ -344,3 +344,8 @@ va-loading-indicator {
     font-size: 20px;
   }
 }
+
+/* resolves css-library removal of !important issue*/
+#edit-preferences-button.vads-u-background-color--transparent {
+  background-color: transparent;
+}
```
Comment (oleksii-morgun) [inline src/applications/mhv-secure-messaging/sass/secure-messaging.scss:351]: tested locally, looks good 👍🏻

**Review by alex-morgun-1 — DISMISSED** [review]
MHV SM looks good ✅

File: src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js [hunk 2]
```diff
@@ -25,14 +25,14 @@ describe('Secure Messaging Verify Links and Buttons Keyboard Nav', () => {
...
-      .find('.docked.reveal', { timeout: 1000 })
-      .should('be.visible');
-    cy.realPress('Tab');
-    cy.get(Locators.ALERTS.BACK_TOP).should('have.focus');
+    // cy.get(Locators.ALERTS.BACK_TOP)
+    //   .shadow()
+    //   .find('.docked.reveal', { timeout: 1000 })
+    //   .should('be.visible');
+    // cy.realPress('Tab');
+    // cy.get(Locators.ALERTS.BACK_TOP).should('have.focus');
```
Comment (mdewey) [inline src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js:35]: QUESTION: Why was this commented out?

**Review by mdewey — COMMENTED** [review]
I didn't run it locally, but the code looks fine, 

1 small question before I can approve

**Review by longmd — DISMISSED** [review]
All good from the `1010-health-apps` side with the `hca`. 🚀

File: src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js (hunk 2 above)
Comment (rmessina1010) [reply src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js:35]: @mdewey, I believe this was Alex that approved the skip of this test.

**Review by mdewey — DISMISSED** [review]

File: src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js (hunk 2 above)
Comment (mdewey) [reply src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js:35]: Sounds good. Thanks @rmessina1010 . 

@oleksii-morgun is this work captured somewhere?

Comment (oleksii-morgun) [reply src/applications/mhv-secure-messaging/tests/e2e/keyboard-nav-test/secure-messaging-keyboard-nav-to-links-buttons-landing-page.cypress.spec.js:35]: @mdewey yes, it's on our radar. @fazilqa has a pending fix that is being tested

**Review by anitadewitt-va — DISMISSED** [review]

**Comment by randimays** [comment]
I reviewed Resources & Support and the header/footer (modernized and injected). Some observations:

**1. On the injected header (va.gov/health), the main (logo) header, `VA Benefits and Health Care` and `About VA` have different background colors than in prod.** This branch on the left, prod on the right.

Screenshot 2024-09-03 at 10 14 33 AM


Screenshot 2024-09-03 at 10 14 38 AM

<hr/>


**2. Color and spacing differences on the modernized header/footer (va.gov). Spacing is different between "Search" and the input box.** This branch on the left, prod on the right.

Screenshot 2024-09-03 at 10 23 09 AM

Screenshot 2024-09-03 at 10 23 14 AM

**Review by randimays — CHANGES_REQUESTED** [review]
Noted some style issues in a comment above.

**Comment by rmessina1010** [comment]
@randimays resolved nav (and crisis banner) color issue for team sites. See test result screenshot. 

Screenshot 2024-09-04 at 5 48 32 PM (2)

**Review by jamigibbs — DISMISSED** [review]

**Review by longmd — DISMISSED** [review]

**Review by alex-morgun-1 — DISMISSED** [review]

**Review by jamigibbs — DISMISSED** [review]

**Review by jamigibbs — DISMISSED** [review]

**Review by randimays — DISMISSED** [review]
Checked header/footer on VA.gov and TeamSites, and also checked `/resources` search. LGTM!

**Review by micahchiang — APPROVED** [review]

**Review by jamigibbs — APPROVED** [review]

**Review by alex-morgun-1 — APPROVED** [review]

**Review by asiisii — APPROVED** [review]
LGTM 🚀

**Review by longmd — APPROVED** [review]

**Review by randimays — APPROVED** [review]
LGTM!

**Review by gabezurita — APPROVED** [review]
✅

**Review by anitadewitt-va — COMMENTED** [review]
Approved.

**Pull request merged**